R/create.R

```diff
@@ -472,75 +472,78 @@ create_obs_model <- function(obs = obs_opts(), dates) {
 #'  example_confirmed, 7, rt_opts(), gp_opts(), obs_opts(), 7,
 #'  backcalc_opts(), create_shifted_cases(example_confirmed, 7, 14, 7)
 #' )
-create_stan_data <- function(reported_cases, seeding_time,
+create_stan_data <- function(data, seeding_time,
                              rt, gp, obs, horizon,
                              backcalc, shifted_cases) {
 
-  cases <- reported_cases[(seeding_time + 1):(.N - horizon)]
+  cases <- data[(seeding_time + 1):(.N - horizon)]
   complete_cases <- create_complete_cases(cases)
   cases <- cases$confirm
 
-  data <- list(
+  stan_data <- list(
     cases = complete_cases$confirm,
     cases_time = complete_cases$lookup,
     lt = nrow(complete_cases),
     shifted_cases = shifted_cases,
-    t = length(reported_cases$date),
+    t = length(data$date),
     horizon = horizon,
     burn_in = 0,
     seeding_time = seeding_time
   )
   # add Rt data
-  data <- c(
-    data,
+  stan_data <- c(
+    stan_data,
     create_rt_data(rt,
-      breakpoints = reported_cases[(data$seeding_time + 1):.N]$breakpoint,
-      delay = data$seeding_time, horizon = data$horizon
+      breakpoints = data[(stan_data$seeding_time + 1):.N]$breakpoint,
+      delay = stan_data$seeding_time, horizon = stan_data$horizon
     )
   )
   # initial estimate of growth
   first_week <- data.table::data.table(
     confirm = cases[seq_len(min(7, length(cases)))],
     t = seq_len(min(7, length(cases)))
   )[!is.na(confirm)]
-  data$prior_infections <- log(mean(first_week$confirm, na.rm = TRUE))
-  data$prior_infections <- ifelse(
-    is.na(data$prior_infections) || is.null(data$prior_infections),
-    0, data$prior_infections
+  stan_data$prior_infections <- log(mean(first_week$confirm, na.rm = TRUE))
+  stan_data$prior_infections <- ifelse(
+    is.na(stan_data$prior_infections) || is.null(stan_data$prior_infections),
+    0, stan_data$prior_infections
   )
-  if (data$seeding_time > 1 && nrow(first_week) > 1) {
+  if (stan_data$seeding_time > 1 && nrow(first_week) > 1) {
     safe_lm <- purrr::safely(stats::lm)
-    data$prior_growth <- safe_lm(log(confirm) ~ t, data = first_week)[[1]]
-    data$prior_growth <- ifelse(is.null(data$prior_growth), 0,
-      data$prior_growth$coefficients[2]
+    stan_data$prior_growth <- safe_lm(log(confirm) ~ t,
+                                      stan_data = first_week
```

Comment:
```suggestion
      stan_data = first_week
```
to conform with package indentation style